user_scripts/wrangler_by_sample.sh

```diff
@@ -29,8 +29,20 @@ function parse_yaml {
    }'
 }
 
+
 eval $(parse_yaml wrangler_by_sample.yaml)
 
+function parse_sample_sheet_directory {
+    readarray -d "/" -t strarr <<< "$input_sample_sheet"
```

Comment:
This is a cool Unix function for splitting up a string into an array. I'll have to try and remember this, thanks for sharing it